libs/libgudev/Makefile

```
@@ -0,0 +1,67 @@
+#
+# Copyright (C) 2024 TDT AG <development@tdt.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See https://www.gnu.org/licenses/gpl-2.0.txt for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libgudev
+PKG_VERSION:=238
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=@GNOME/libgudev/$(basename $(PKG_VERSION))
+PKG_HASH:=61266ab1afc9d73dbc60a8b2af73e99d2fdff47d99544d085760e4fa667b5dd1
+
+PKG_MAINTAINER:=Lukas Voegl <lvoegl@tdt.de>
+PKG_LICENSE:=LGPL-2.1-or-later
+PKG_LICENSE_FILES:=COPYING
+
+PKG_BUILD_DEPENDS:=glib2/host
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+include $(INCLUDE_DIR)/meson.mk
+
+define Package/libgudev
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=libgudev
+  URL:=https://wiki.gnome.org/Projects/libgudev
```

I checked and that website is gonna be retired soon. What I have checked is that the official website is their GitLab.
```suggestion
  URL:=https://gitlab.gnome.org/GNOME/libgudev
```